readme.md

```diff
@@ -25,6 +25,8 @@ $ npm install --save pwmetrics
 #              Median run selected by run with the median TTI.
 pwmetrics http://example.com/ --runs=3
 
+# --submit     Submits results to a google spreadsheet (requires some setup)
+pwmetrics http://goat.com --submit
```

Comment:
I imagine we'll have a config object somewhere that can have clientsecret/api-token as well as indicate the endpoint.